website/tables/aws/aws_applicationautoscaling_scheduled_actions.md

```diff
@@ -0,0 +1,30 @@
+# Table: aws_applicationautoscaling_scheduled_actions
+
+This table shows data for Applicationautoscaling Scheduled Actions.
```

Comment:
Please add another exception for `Applicationautoscaling` to plugin.go for the docs generation 😄

Reply:
oof it reindented everything so I added it to the very top